### PR TITLE
chore(github): Code Owners of devfile registry is members of Language…

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?

Based on last che SOS and based on repositories scope:

https://docs.google.com/document/d/1h6t4novpcRHjxe8DrEhTamDmit9VviOqcMQNtahM9uE/edit#heading=h.nh4qax6oermn

owners of che-devfile-registry is Thomas with languages team

Change-Id: I3e67500e8306a6b6ff848cdf892a86bd35a057fd
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
I would keep both @amisevsk & @nickboldt on the list as we do a lot of code review here too

**Comment by benoitf** [comment]
why it has been modified before merging ?

**Comment by nickboldt** [comment]
There was some disagreement about the value of removing me and Angel, and no one from the new Lang team objected so I fixed the PR and merged it. :)

Also until Eric is voted in as a committer, it makes sense to me to have a couple backup owners in his TZ who can push commits on his behalf.

If you want to instead fix the repo to allow non-codeowners to merge, then there's little point in having a codeowners file, in which case I'd have the same permissions you plan to bestow on my by making the repo config change.

TL;DR, is there a problem?

**Comment by benoitf** [comment]
> Also until Eric is voted in as a committer, it makes sense to me to have a couple backup owners in his TZ who can push commits on his behalf.

Sorry but Eric is not owner of devfile registry.

> If you want to instead fix the repo to allow non-codeowners to merge, then there's little point in having a codeowners file, in which case I'd have the same permissions you plan to bestow on my by making the repo config change.

you can already merge without being code owners on this repository which is the default rule for all Eclipse Che repositories (except plugin registry but will be fixed)
